### ISSUE TITLE
Demo not working correctly

### ISSUE THREAD

**Issue by lim169** [peterjurkovic/pjTextToSpeech#2]
It takes only from cache example "Hello world" works, but when tried "start", it never have loaded.
Possible that google blocked something.

**Comment by peterjurkovic** [comment]
Yes, I am aware of this. The module uses free Google Text-to-Speach API which was switched off couple of months ago. Google introduced a new payed version of TTS api which has not been integrated yet.

**Comment by AdityaParashar93** [comment]
angular.js:8561 GET http://tts.dev/tts-backend/index.php?lang=en&text=This+is+login+controller+and+we+welcome+you+to+test+it. net::ERR_ICANN_NAME_COLLISION

Getting this error. Any Idea what its about...?

Trying to use TTS for making website for blind people....any help would be appreciated.

**Comment by peterjurkovic** [comment]
@AdityaParashar93  I would recommend take a look on https://superuser.com/questions/919278/err-icann-name-collision-when-trying-to-use-localhost-dev-in-chrome
